widgets/common/src/deployModal/DeployModalActions.tsx

```diff
@@ -0,0 +1,94 @@
+import { FunctionComponent } from 'react';
+
+const t = Stage.Utils.getT('widgets.common.deployments.deployModal');
+
+interface ApproveButtonsProps {
+    loading: boolean;
+    showDeployButton: boolean;
+    onInstall: () => void;
+    onDeploy: () => void;
+}
+
+const ApproveButtons: FunctionComponent<ApproveButtonsProps> = ({ showDeployButton, onInstall, loading, onDeploy }) => {
+    enum Buttons {
+        install,
+        deploy
+    }
+    const { ApproveButton, Button, Dropdown } = Stage.Basic;
+    const { useInput } = Stage.Hooks;
+    const [selectedButton, setSelectedButton] = useInput(Buttons.install);
+
+    if (!showDeployButton) {
+        <ApproveButton
+            onClick={onInstall}
+            disabled={loading}
+            content={t('buttons.deployAndInstall')}
+            icon="cogs"
+            className="green"
+        />;
+    }
+    return (
+        <Button.Group color="green">
+            {selectedButton === Buttons.install ? (
+                <ApproveButton onClick={onInstall} disabled={loading} content={t('buttons.install')} icon="cogs" />
+            ) : (
+                <ApproveButton onClick={onDeploy} disabled={loading} content={t('buttons.deploy')} icon="rocket" />
+            )}
+
+            <Dropdown
+                className="icon"
+                button
+                clearable={false}
+                floating
+                aria-label="Deploy or Install"
```

Comment:
Optional:

In most of the cases it is reasonable to have alphabetical order IMHO:

```suggestion
                aria-label="Deploy or Install"
                button
                className="icon"
                clearable={false}
                floating
```

but not to block this one - can be part of the next PR related to Create Deployment modal.

Reply:
👍🏽